libraries/standard/mqtt/test/system/iot_tests_mqtt_system.c

```diff
@@ -220,6 +227,44 @@ static bool _disconnectSerializerOverride = false;  /**< @brief Tracks if #_disc
 
 /*-----------------------------------------------------------*/
 
+/**
+ * @brief Establish an MQTT connection. Retry if enabled.
+ */
+static IotMqttError_t _mqttConnect( const IotMqttNetworkInfo_t * pNetworkInfo,
+                                    const IotMqttConnectInfo_t * pConnectInfo,
+                                    uint32_t timeoutMs,
+                                    IotMqttConnection_t * const pMqttConnection )
+{
+    IotMqttError_t status = IOT_MQTT_STATUS_PENDING;
+
+    int32_t retryCount = 0;
+
+    /* AWS IoT Service limits only allow 1 connection per MQTT client ID per second.
+     * Wait until 1100 ms have elapsed since the last connection. */
+    uint32_t periodMs = 1100;
+
+    for( ; retryCount <= IOT_TEST_MQTT_CONNECT_RETRY_COUNT; retryCount++ )
```

Comment:
Needs to be `retryCount < IOT_TEST_MQTT_CONNECT_RETRY_COUNT`

Reply:
Done thanks